bin/blazegraph_load_api.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# On a debian package installation, the original file could be found at:
+#/usr/bin/loadRestAPI.sh
+
+if [ "$1" == "" -o "$1" == "-h" -o "$1" == "--help" ]; then
+    echo "Usage: $0 {file_or_directory} [{graph_namespace}]"
+    exit
+else
+    FILE_OR_DIR=$1
+fi
+
+if [ ! -z "$2" ]; then
+    NAMESPACE=$2
+else
+    NAMESPACE=kb
+fi
+
+[ -f /etc/default/blazegraph ] && source /etc/default/blazegraph
+[ -z "$JETTY_PORT" ] && JETTY_PORT=9999
```

Comment:
Blazegraph can be deployed in various ways and on my laptop, it does have the `/etc/default/blazegraph` file.  This code checks for this config file and, if the JETTY_PORT is not set, then it sets it to 9999 (the default port).  In this code base, these are just default settings, they do not apply to any deployment system in particular - for that, look at the shared_configs data for the box concerned.